py/py3/fit.py

```diff
@@ -29,7 +29,7 @@
 oc.set_fit_model(sys.argv[2])
 oc.set_action(Action.FIT)
 oc.set_report_separator(SeparatorType.SPACE)
-oc.set_skip_nominal(True)
+oc.set_skip_nominal(1)
```

Comment:
This is True

Reply:
I normalized all the calls to use `int` rather than `True`, since that's what the rest of the calls to this function were already using anyway.